packages/wonder-blocks-grid/components/row.js

```diff
@@ -46,12 +44,7 @@ export default class Row extends React.Component<{
      * Can also accept a function which receives the `gridSize` and
      * `totalColumns` and should return some React Nodes to render.
      */
-    children:
```

Comment:
Oh, what was wrong with this? I felt like this would still be helpful as it would define the arguments for the callback function.

Reply:
This wasn't necessary.  There were lots of flow errors when I started on this diff and I misinterpreted some of them.  Thanks for catching this.  Always better to have better types.